app/main.py

```diff
@@ -1,2 +1,40 @@
 class Dictionary:
-    pass
+    def __init__(self, initial_capacity: int = 8) -> None:
+        self.capacity = initial_capacity
+        self.size = 0
+        self.table = [None] * self.capacity
+
+    def __setitem__(self, key: object, value: object) -> None:
+        if self.size >= self.capacity * 0.75:
```

Comment:
The resizing condition `self.size >= self.capacity * 0.75` is correct for triggering a resize when the load factor exceeds 0.75. However, ensure that this condition aligns with the task requirements if specified differently.